platform/lib/routers/example/sources.js

```diff
@@ -18,21 +18,24 @@
 
 const express = require('express');
 const utils = require('@lib/utils');
+const exampleBackend = require('@examples');
 
 const SOURCES_DEST = utils.project.absolute('/dist/examples/sources/');
 
 // eslint-disable-next-line new-cap
 const exampleSources = express.Router();
 const staticSources = express.static(SOURCES_DEST);
 
-exampleSources.use('/examples/:category/:name/:snippetId?', (request, response, next) => {
-  request.url = `/${request.params.category}/${request.params.name}`;
-  if (request.params.snippetId) {
-    request.url += `-${request.params.snippetId}`;
+exampleSources.use('/documentation/examples/', exampleBackend);
```

Comment:
I remember you mentioning this in our hangout, seeing it written makes the dilemma obvious at all 😅Maybe registering the sources router with `.get` and the route instead of `.use` in `platform.js` could separate things again.